packages/ui-components/src/index.js

```diff
@@ -5,7 +5,7 @@
 
 import "./global.scss"
 export { AppBody } from "./components/AppBody/AppBody.component"
-export { AppIntro } from "./components/AppIntro/AppIntro.component"
+export { AppIntro } from "./deprecated_js/AppIntro/AppIntro.component"
```

Comment:
Maybe we could group all deprecated imports not alphabatically just under one comments line and put everything there WDYT?

Reply:
Nice suggestion, done.